sensor_msgs/Image hst
int32 dim
---
int32[] res
